modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/definitions/portlet/action/test/GetObjectDefinitionInfoMVCResourceCommandTest.java

```diff
@@ -0,0 +1,148 @@
+import com.liferay.object.test.util.ObjectDefinitionTestUtil;
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.portlet.PortletConfigFactoryUtil;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCResourceCommand;
+import com.liferay.portal.kernel.service.CompanyLocalService;
+import com.liferay.portal.kernel.service.PortletLocalService;
+import com.liferay.portal.kernel.service.WorkflowDefinitionLinkService;
+import com.liferay.portal.kernel.test.portlet.MockLiferayResourceRequest;
+import com.liferay.portal.kernel.test.portlet.MockLiferayResourceResponse;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.test.util.ServiceContextTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.JavaConstants;
+import com.liferay.portal.kernel.util.WebKeys;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+import com.liferay.portal.workflow.constants.WorkflowDefinitionConstants;
+import com.liferay.portal.workflow.kaleo.service.KaleoDefinitionLocalService;
+
+import java.io.ByteArrayOutputStream;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Nathaly Gomes
+ */
+@RunWith(Arquillian.class)
+public class GetObjectDefinitionInfoMVCResourceCommandTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Test
+	public void testGetObjectDefinitionInfo() throws Exception {
+		ObjectDefinition objectDefinition =
+			ObjectDefinitionTestUtil.addCustomObjectDefinition();
+
+		String workflowDefinitionName = RandomTestUtil.randomString();
+
+		_kaleoDefinitionLocalService.addKaleoDefinition(
+			RandomTestUtil.randomString(), workflowDefinitionName,
+			workflowDefinitionName, null, null,
+			WorkflowDefinitionConstants.SCOPE_ALL, 1,
+			ServiceContextTestUtil.getServiceContext());
+
+		_workflowDefinitionLinkService.addWorkflowDefinitionLink(
+			TestPropsValues.getUserId(), TestPropsValues.getCompanyId(),
+			TestPropsValues.getGroupId(), objectDefinition.getClassName(), 0, 0,
+			workflowDefinitionName, 1);
+
+		JSONObject jsonObject = _getObjectDefinitionInfoJSONObject(
+			objectDefinition.getObjectDefinitionId());
+
+		Assert.assertEquals(
+			JSONUtil.put(
+				"tableName", objectDefinition.getDBTableName()
+			).put(
+				"workflowDefinitionLabel", workflowDefinitionName
```

Comment:
It would be nice to match these strings here, like:

```
"workflowDefinitionTitle", workflowDefinitionTitle
"workflowDefinitionLabel", workflowDefinitionLabel
"workflowDefinitionName", workflowDefinitionName
```